src/ThemeToggle.svelte

```diff
@@ -0,0 +1,187 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+
+  let theme: "light" | "dark" | undefined = undefined;
+
+  onMount(() => {
+    let storedTheme = window.localStorage.getItem('theme');
```

Comment:
IMO we should move the page theme to the `session` store and have some store functions there to get and set the current theme.

Reply:
Couldn't move it to `session`, because it's only available when you've connected your wallet.
How about this: 2b1adbfae38338e4f7bd93457e16f40ab2eb8ae5? Could also get a second opinion @cloudhead?